.github/workflows/reusable_swap_functional_tests.yml

```diff
@@ -68,7 +68,7 @@ on:
 
       branch_for_solana:
         required: false
-        default: 'develop'
+        default: 'fix/LIVE-11792'
```

Comment:
Added this in a separate commit that I can remove when we merge the fix on app-solana

Reply:
👍